File: Habarovsk/Bars.GkhGji.Regions.Habarovsk/libs/B4/model/appealcits/AppealCitsResolution.js
Ext.define('B4.model.appealcits.AppealCitsResolution', {
    extend: 'B4.base.Model',
    idProperty: 'Id',
    proxy: {
        type: 'b4proxy',
        controllerName: 'AppealCitsResolution'
    },
    fields: [
        { name: 'Id' },
        { name: 'AppealCits' },
        { name: 'ResolutionText' },
        { name: 'ResolutionTerm' },
        { name: 'ResolutionAuthor' },
        { name: 'ResolutionDate' },
        { name: 'ResolutionContent' },
        { name: 'ParentResolutionData', defaultValue: null },
        { name: 'ImportId'},
        { name: 'ParentId'},
        { name: 'Executed'}
    ]
});

File: Voronezh/Bars.GkhGji.Regions.Voronezh/Entities/AppealCits/AppealCitsResolution.cs
namespace Bars.GkhGji.Regions.Voronezh.Entities
{
    using System;
    using Bars.B4.Modules.FileStorage;
    using Bars.Gkh.Entities;
    using Bars.Gkh.Enums;
    using Bars.GkhGji.Entities;

    /// <summary>
    /// Резолюция АС ДОУ
    /// </summary>
    public class AppealCitsResolution : BaseGkhEntity
    {
        /// <summary>
        /// Обращение граждан
        /// </summary>
        public virtual AppealCits AppealCits { get; set; }

        /// <summary>
        /// Текст резолюции
        /// </summary>
        public virtual string ResolutionText { get; set; }

        /// <summary>
        /// Срок резолюции
        /// </summary>
        public virtual DateTime ResolutionTerm { get; set; }

        /// <summary>
        /// Автор резолюции
        /// </summary>
        public virtual string ResolutionAuthor { get; set; }

        /// <summary>
        /// Дата резолюции
        /// </summary>
        public virtual DateTime ResolutionDate { get; set; }

        /// <summary>
        /// Содержание резолюции, включая текст.
        /// </summary>
        public virtual string ResolutionContent { get; set; }

        /// <summary>
        /// ID АС ДОУ
        /// </summary>
        public virtual string ImportId { get; set; }

        /// <summary>
        /// Родитель (ID АС ДОУ)
        /// </summary>
        public virtual string ParentId { get; set; }

        /// <summary>
        /// отчет принят
        /// </summary>
        public virtual YesNoNotSet Executed { get; set; }
    }
}